bazel/repositories.bzl

```diff
@@ -1037,14 +1026,7 @@ def _proxy_wasm_cpp_sdk():
     external_http_archive(name = "proxy_wasm_cpp_sdk")
 
 def _proxy_wasm_cpp_host():
-    external_http_archive(
-        name = "proxy_wasm_cpp_host",
-        # proxy-wasm-cpp-host-s390x-support.patch fixes WASM on s390x error https://issues.redhat.com/browse/OSSM-1815
-        # maistra-2.3 issue: https://issues.redhat.com/browse/OSSM-1956
-        # The permanent fix is: https://github.com/proxy-wasm/proxy-wasm-cpp-host/pull/282
-        patches = ["@envoy//bazel/external:proxy-wasm-cpp-host-s390x-support.patch"],
-        patch_args = ["-p1"],
```

Comment:
Are you sure about this? To me this was the most important change I commented in PR 225. The others were most likely just nit's.

Reply:
Yes, I discussed this with Konstantin.

The dependency is has been updated upstream, and Konstantin indicates the patch he needs is in place.

Reply:
Great. Then I'm assuming the upstream changes are present in `624ef2edac2cdc086e0339f36b32ef1a2ed7e270` which is the commit we are pulling in in repository_locations.bzl, right?

cc @knm3000

Reply:
Awesome! Thanks for checking this @knm3000. And sorry for the noise @twghu